### PR TITLE
Update delete lesson and add lesson command details in user guide

### PR DESCRIPTION
**Pull request by eezj35**
Things updated for add lesson and delete lesson
- command word
- command information
- examples
- command summary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -109,8 +109,8 @@ Format: `add -l s/SUBJECT g/GRADE d/DAY_OF_WEEK t/TIME_START c/COST`
 </div>
 
 Examples:
-* `add -l s/Science g/P5 d/Wed t/1230 c/12.0`
-* `add -l s/Mathematics g/S4 d/Fri t/1500 c/10.3`
+* `add-l s/Science g/P5 d/Wed t/12:30 c/12.0`
```
Comment (Timothyoung97) [inline docs/UserGuide.md]: I think for the time, we're leaving it as 1200 without the `:`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -109,8 +109,8 @@ Format: `add -l s/SUBJECT g/GRADE d/DAY_OF_WEEK t/TIME_START c/COST`
...
+* `add-l s/Mathematics g/S4 d/Fri t/15:00 c/10.3`
```
Comment (michael-lee-sk) [inline docs/UserGuide.md:113]: I think it may be t/1230 not t/12:30. Likewise for 1500

File: docs/UserGuide.md [hunk 3]
```diff
@@ -231,11 +231,11 @@ If your changes to the data file makes its format invalid, TuitiONE will discard
 Action | Format, Examples
 --------|------------------
 **Add** | `add n/NAME p/PARENT_PHONE_NUMBER e/EMAIL a/ADDRESS g/GRADE [t/TAG]…` <br> e.g. `add n/Betsy Crowe p/91234567 e/bc@gmail.com a/Bleecker street, block 123, #01-01 g/S5 t/foreign student`
-**Add lesson** | `add -l s/SUBJECT g/GRADE d/DAY_OF_WEEK t/TIME_START c/COST` <br> e.g. `add -l s/Science g/P5 d/Wed t/12:30 c/12.0`
+**Add lesson** | `add-l s/SUBJECT g/GRADE d/DAY_OF_WEEK t/TIME_START c/COST` <br> e.g. `add-l s/Science g/P5 d/Wed t/12:30 c/12.0`
```
Comment (michael-lee-sk) [inline docs/UserGuide.md:234]: Likewise for this, should be t/1230 not t/12:30

**Review by Timothyoung97 — APPROVED** [review]
LGTM

**Review by michael-lee-sk — APPROVED** [review]

**Pull request merged**